test/functional/lib/tests.cpp

```diff
@@ -2350,7 +2438,9 @@ GenericRxMatch(
     Rule.Action = XDP_PROGRAM_ACTION_PASS;
 
     ProgramHandle =
-        CreateXdpProg(If.GetIfIndex(), &XdpInspectRxL2, If.GetQueueId(), XDP_GENERIC, &Rule, 1);
+        CreateXdpProg(
+            If.GetIfIndex(), &XdpInspectRxL2, If.GetQueueId(), XDP_GENERIC, &Rule, 1,
+            XDP_CREATE_PROGRAM_FLAG_ALL_QUEUES);
```

Comment:
Why are we setting the `ALL_QUEUES` flag here? It's still important to support and test per-queue programs for the time being.

Reply:
Removed